defaults/main.yml

```diff
@@ -27,6 +27,11 @@ funkwhale_media_path: "{{ funkwhale_data_path }}/media"
 # The `funkwhale_music_path` defines a place where you can put music that can then be imported into funkwhale.
 funkwhale_music_path: "{{ funkwhale_data_path }}/music"
 
+# Enables automatic in-place import and watching file and metadata changes in the funkwhale_music_path directory.
+# Needs the first 8 characters of the library ID after creating a library in the web UI. If your library ID is "769a2bc3-eb1d-4aff-9f84-2c4d80d5c2d1", provide "769a2bc3".
+funkwhale_watch_imports_enabled: false
+funkwhale_watch_imports_library_id: ""  # add first 8 characters of library ID here
```

Comment:
Is it possibly for the watcher script to potentially accommodate watching multiple library ids?

Could a single `import_files` call deal with multiple ids?

Reply:
From funkwhale's [import music documentation](https://docs.funkwhale.audio/administrator/import.html#import-your-music) it is not clear whether this is possible with a single command.

From `funkwhale-manage import_files --help` it seems like only one local library can be imported at once:
```
positional arguments:
  library_id            A local library identifier where the files should be
                        imported. You can use the full uuid such as
                        e29c5be9-6da3-4d92-b40b-4970edd3ee4b or only a small
                        portion of it, starting from the beginning, such as
                        e29c5be9
  path
```

One could certianly specify different libraries for different paths and spawn a dedicated service for each one of them. This would need some more work, as one would need to create docker mount points for each local path (preferrably under `funkwhale_music_path`) and associate each mount point with a different library ID.